### PR TITLE
style(*): Use hsla for surface to apply proper background when theme is changed

### PR DESCRIPTION
**Pull request by dkamburov**

### CODE REVIEW

File: projects/app-lob/src/app/index/docs-layout.component.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ::ng-deep {
 	.custom-body {
-		color: var(--igx-surface-500-contrast);
-		background: var(--igx-surface-500);
+		color: hsla(var(--igx-surface-500-contrast));
```
Comment (wnvko) [inline projects/app-lob/src/app/index/docs-layout.component.scss:3]: `hsla` function should not be applied to contrast colors

File: src/app/index/docs-layout.component.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ::ng-deep {
 	.custom-body {
-		color: var(--igx-surface-500-contrast);
-		background: var(--igx-surface-500);
+		color: hsla(var(--igx-surface-500-contrast));
```
Comment (wnvko) [inline src/app/index/docs-layout.component.scss:3]: `hsla` function should not be applied to contrast colors

**Review by wnvko — CHANGES_REQUESTED** [review]

**Review by zdrawku — CHANGES_REQUESTED** [review]
@simeonoff @wnvko @dkamburov 

Why we dont apply hsla to contrast colors?

Also, can we update all css palette variables to use the hsla css function

image

**Comment by simeonoff** [comment]
> @simeonoff @wnvko @dkamburov
> 
> Why we dont apply hsla to contrast colors?
> 
> Also, can we update all css palette variables to use the hsla css function
> 
> image

Contrast colors are always either black or white - no need. No, the way it is right now allows you to pass the values to either the HSL or the HSLA function.

**Review by simeonoff — DISMISSED** [review]

**Review by wnvko — DISMISSED** [review]

**Comment by zdrawku** [comment]
@dkamburov does the problem with the backgroud apper to ther sample as well, not only the docs-layout, but toggle, calendar and multiview?

image

**Comment by dkamburov** [comment]
> @dkamburov does the problem with the backgroud apper to ther sample as well, not only the docs-layout, but toggle, calendar and multiview?
> 
> image

This change is related to the background of all samples but if you are ok to I can change the surface for those specific sample styles as well.

**Comment by zdrawku** [comment]
> > @dkamburov does the problem with the backgroud apper to ther sample as well, not only the docs-layout, but toggle, calendar and multiview?
> > image
> 
> This change is related to the background of all samples but if you are ok to I can change the surface for those specific sample styles as well.

If the same problem persists there as well, please do the change, otherwise just dont :)

**Pull request merged**